Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb

```diff
@@ -36,9 +36,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 33,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'fbprophet'",
```

Comment:
You don't have a virtual environment set up for others so the file will always be broken to others